protobuf/daemon/account.proto

```diff
@@ -4,9 +4,15 @@ package pb;
 
 option go_package = "github.com/NordSecurity/nordvpn-linux/daemon/pb";
 
-message DedidcatedIPServices {
-  int64 dedicated_ip_status = 5;
-  string dedicated_ip_expires_at = 6;
+message DedidcatedIPService {
+  int64 server_id = 1;
+  string dedicated_ip_expires_at = 2;
+}
+
+enum MFAStatus {
```

Comment:
wouldn't make sense to move this enum in `common.proto` rename to something more generic, because it might be useful in the future for other responses?
possible names :melting_face:  `TriState`, `State`, `TriBool`...?